fdbrpc/HTTP.actor.cpp

```diff
@@ -705,15 +705,17 @@ ACTOR Future<Void> sendProxyConnectRequest(Reference<IConnection> conn,
 	state double nextRetryDelay = 2.0;
 	state Reference<IRateControl> sendReceiveRate = makeReference<Unlimited>();
 	state int64_t bytes_sent = 0;
+	state UnsentPacketQueue empty_packet_queue;
 
 	state Reference<HTTP::OutgoingRequest> req = makeReference<HTTP::OutgoingRequest>();
 	req->verb = HTTP_VERB_CONNECT;
 	req->resource = remoteHost + ":" + remoteService;
-	req->data.content = nullptr;
+	req->data.content = &empty_packet_queue;
```

Comment:
Another option would be to check in the `doRequest` actor if the content is a `nullptr` and if so set the empty packet queue.